### DISCUSSION TITLE
about v2.3.x "Merged IndexCoord and DataCoord",

### DISCUSSION THREAD

**Discussion by xiaozhuang-a** [milvus-io/milvus#28613] [Q&A and General discussion]
i found，milvus v2.3.x  version have feation about "Merged IndexCoord and DataCoord", but use helm When deploying, there were still no changes compared to before, both components were deployed synchronously

**Comment by xiaofan-luan** [comment]
Good catch.
The index coord is a fake to avoid k8s reboot while upgrade.

we will remove index coord process later in the future
